Makefile

```diff
@@ -0,0 +1,27 @@
+
+cs:
+	./vendor/bin/phpcs --standard=PSR2 --report=emacs --extensions=php --warning-severity=0 src/ tests/PHPSA/
+
+check-src:
+	./bin/phpsa check -vvv ./src
+
+# Rubric, hacking themselves (c) @Kistamushken
+check-fixtures:
+	./bin/phpsa check -vvv ./fixtures
+
+# For local dev
+dev:
+	./bin/phpsa check -vvv ./sandbox
+
+tests-local:
+	./vendor/bin/phpunit -v
+
+# Alias for tests-local
+tests: tests-local
+
+tests-ci:
+	./vendor/bin/phpunit -v --debug --coverage-clover=coverage.clover
+
+test: cs tests-local check-fixtures check-src
```

Comment:
@ddmler 
I didnt understand what you want to suggest, can you create a diff or PR to this PR?
Thanks